apps/browser/components/ChatLogs.tsx

```diff
@@ -62,7 +63,7 @@ export default function ChatLogs(props: ChatLogsProps) {
   return (
     <>
       <div className="flex h-20 items-center justify-center border-b-2 border-zinc-800 md:h-12">
-        {/* <div>{chatName}</div> */}
+        <div>{chatName && chatName}</div>
```

Comment:
Should we have a default value here? Can we make the `chatName` required?

Reply:
i will update it to receive the prop always as string (with the `chatName || "New Session"` you mentioned in the other comment)